packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -601,6 +601,21 @@ export class CeramicCliUtils {
   }
 }
 
+/**
+ * Helper function: Parse provided port and verify validity or exit process
+ * @param inPort
+ */
+export function validatePort(inPort) {
```

Comment:
now that you only use this function in ceramic-daemon.ts, you can just put it there and not need to export it.